python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -23,7 +24,17 @@
 
 class RankingConstants:
     NEW_RANK = "new_rank"
+    NEW_MRR = "new_MRR"
+    OLD_MRR = "old_MRR"
+    OLD_RANKNMF = "old_RankNMF"
+    NEW_RANKNMF = "new_RANKNMF"
+    MRR_DIFF = "MRR_diff"
     TTEST_PVALUE_THRESHOLD = 0.1
+    STATISTICAL_POWER = 0.9
+    # list of metrics for which to compute the variance in batches.
+    VARIANCE_METRIC_LIST = [NEW_MRR, OLD_MRR]
+    # list of metrics for which power analysis to be used.
+    METRIC_POWER_ANALYSIS = ["MRR"]
```

Comment:
These need to be moved to a model config and not hardcoded. I think an even cleaner/better thing to do is define an `evaluation_config.yaml`. That makes the feature_config, model_config and eval_config as clean loosely connected pieces to control the model run.